### PR TITLE
WIP: Fix stream name

### PR DESCRIPTION
**Pull request by danielballan**
One more instance of "maybe Ken was right," for the road.

This is WIP, needs more tests.

### CODE REVIEW

File: databroker/core.py [hunk 1]
```diff
@@ -22,6 +22,11 @@
 logger = logging.getLogger(__name__)
 
 
+class ALL:
```
Comment (tacaswell) [inline databroker/core.py]: Is this pattern better than `ALL = object()`?

Comment (klauer) [inline databroker/core.py]: Isn't it just so he can shove the docstring in there?

Comment (tacaswell) [inline databroker/core.py]: never mind, this is the pattern numpy follows.

**Comment by danielballan** [comment]
This passes all pymongo-based tests but fails sqlite because monitors emit documents from a separate thread. That is a problem with portable-mds, not databroker. Effectively, we are requiring `insert_*` methods to be threadsafe, and portable-mds needs to be fixed to satisfy that.

**Comment by tacaswell** [comment]
That is a fun bug.  The thread-non-safeness of sqlite has caused us a fair amount of pain.

**Pull request merged**